source/ui/Components/Output/LightstripSettingsWindow.qml

```diff
@@ -106,8 +115,12 @@ Window {
                 // Reads the current input and passes the new value to the model.
                 function processInput() {
                     var inputValue = controlsColumn.integerFromInputString(String(numberOfStripsTextField.text));
-                    if (inputValue !== undefined && inputValue > 0) {
+                    var size = outputComponent.getMemorySize();
+                    var maxStrips = (outputComponent.getMemorySize() - size) * 8;
```

Comment:
`maxStrips` will always be 0, as you calculate `(outputComponent.getMemorySize() - outputComponent.getMemorySize()) * 8`. Guess it should be `size - controlsColumn.integerFromInputString(String(baseAddressTextField.text))`.